students/12th/eunsulim/style/login.css

```
@@ -0,0 +1,280 @@
+* { 
+    list-style-type: none;
+    text-decoration: none;
+    padding: 0;
+    margin: 0;
+    box-sizing:border-box;
+}
+
+html, body {
+    height:100%;
+    background-color: #fafafa;
+}
+
+.wrap {
+    display: flex;
+    display: -webkit-flex;
+    justify-content: center;
+    height: 100%;
+}
+
+.container {
+    display: flex;
+    display: -webkit-flex;
+    flex-direction: column;
+    flex: 1;
+    -webkit-flex: 1;
+}
+
+.content {
+    display: flex;
+    display: -webkit-flex;
+    -webkit-box-flex: 1;
+    -webkit-flex-grow: 1;
+    -ms-flex-positive: 1;
+    flex-grow: 1;
+}
+
+.inner_box{
+    display: flex;
+    display: -webkit-flex;
+    justify-content: center;
+    align-items: center;
+    margin: 30px auto 0 ;
+    padding-bottom: 44px;
+    max-width: 935px;
+    width: 100%;
+}
+
+.slider_box {
+    background-image: url(../img/phone_img.png);
+    background-size: cover;
+    width: 454px;
+    height: 618px;
+}
+
+.slider_img {
+    position: relative;
+    margin: 99px 0 0 151px;
```

```suggestion
    margin-top: 99px;
    margin-let: 151px;
```
이 경우 그냥 이렇게 써주는게 어떨까요?

---

네 그렇게 수정 했습니다.